motioneye/static/js/main.js

```diff
@@ -1318,23 +1317,34 @@ function enableMaskEdit(cameraId, width, height) {
 
     var selectedCameraId = $('#cameraSelect').val();
     if (selectedCameraId && (!cameraId || cameraId == selectedCameraId)) {
-        $('#saveMaskButton, #clearMaskButton').css('display', 'inline-block');
-        $('#editMaskButton').css('display', 'none');
+        $('#'+maskClass+'MaskSaveButton, #'+maskClass+'MaskClearButton').css('display', 'inline-block');
+        $('#'+maskClass+'MaskEditButton').css('display', 'none');
     }
     
     if (!overlayVisible) {
         showCameraOverlay();
     }
 }
 
-function disableMaskEdit(cameraId) {
-    var cameraFrames;
-    if (cameraId) {
```

Comment:
disableMaskEdit never seemed to get called with a cameraId, so I removed related parts from the code.